test/integration_test/utils/storagecluster.go

```diff
@@ -88,6 +92,22 @@ func UpdateStorageCluster(cluster *corev1.StorageCluster) (*corev1.StorageCluste
 	return operator.Instance().UpdateStorageCluster(cluster)
 }
 
+// UpdateAndValidateStorageCluster update and validate cluster
+func UpdateAndValidateStorageCluster(cluster *corev1.StorageCluster, pxSpecImages map[string]string, t *testing.T) {
+	logrus.Infof("Update StorageCluster %s in %s", cluster.Name, cluster.Namespace)
+
+	cluster, err := operator.Instance().UpdateStorageCluster(cluster)
```

Comment:
can we return the updated cluster for this function?

Reply:
Added getting cluster at the end of UpdateAndValidateStorageCluster() and returning new liveCluster, it seems that UpdateStorageCluster() function doesn't return the very last recent version of StorageCluster object